.github/workflows/ci.yaml

```
@@ -10,10 +10,10 @@ jobs:
       strategy:
         matrix:
           os: [macos-latest, ubuntu-22.04, windows-latest]
-          python: ['3.7', '3.8', '3.9', '3.10']
+          python: ['3.8', '3.9', '3.10', '3.11']
```

Might as well add 3.12 as well.

```suggestion
          python: ['3.8', '3.9', '3.10', '3.11', '3.12']
```